### PR TITLE
feat: Add ProductCard and SwatchBar components

### PR DESCRIPTION
**Pull request by anandgorantala**

### CODE REVIEW

**Review by joerideg — DISMISSED** [review]
Looks good to me. I do think the compound component pattern works great here. 

Could use some tests but lets say we will add those soon :)

**Review by joerideg — CHANGES_REQUESTED** [review]
Requesting that you remove the price formatting part.

File: src/utils/format-price.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const formatPrice = (price: number, currency: string) => {
```
[the remark this answers is not in the document]
Comment (anandgorantala) [reply src/utils/format-price.ts:1]: Good catch, I missed the locale  I had this function in every one of the code samples, so thought this, along with the component would be handy to have as part of the UI Kit.  wdyt about making the locale also an argument?

[the remark this answers is not in the document]
Comment (joerideg) [reply src/utils/format-price.ts]: "It's still optional to use, customers can choose to not import this if they have their own formatter"
Yes I think if its optional and configurable (maybe check if the format prop is passed?) it should be a good utility

**Review by joerideg — APPROVED** [review]

**Pull request merged**